internal/manager/config.go

```diff
@@ -105,6 +106,7 @@ func (c *Config) FlagSet() *pflag.FlagSet {
 	flagSet.StringVar(&c.KongAdminAPIConfig.CACert, "kong-admin-ca-cert", "", `PEM-encoded CA certificate to verify Kong's Admin SSL certificate.`)
 	flagSet.StringSliceVar(&c.KongAdminAPIConfig.Headers, "kong-admin-header", nil, `add a header (key:value) to every Admin API call, this flag can be used multiple times to specify multiple headers`)
 	flagSet.StringVar(&c.KongAdminToken, "kong-admin-token", "", `The Kong Enterprise RBAC token used by the controller.`)
+	flagSet.StringVar(&c.KongNonAdminToken, "kong-non-admin-token", "", `The Kong Enterprise RBAC non-admin user token used by the controller.`)
```

Comment:
question: can you help me to understand why this needs to be a public facing flag? I feel like I'm missing some context here.

Reply:
First, we should understand kong-admin-token works as exactly it named - it accept kong-admin user token because KIC use it access some endpoints (i.e workspaces) which requests admin role permission.
And issue 1279 purposely request that KIC should be able to communicate to Kong with non-admin rbac user, which justify this interface (for non-admin user only ) on top of a non-default workspace.

Reply:
I think what's puzzling me here is that while there's only a single "admin" user, there can be multiple "non-admin" users, correct? Doesn't the potential for any number of non-admin users make this flag seem a little strange? Would it be better to call this `kong-controller-user-token` or something like that, to indicate that this is for a specific non-admin user that the controller will utilize?

Reply:
The original `--kong-admin-token` argument is sufficient for what we need, and we should not add this additional flag/associated behavior. For #1279 we just want integration testing of the existing `--kong-admin-token` and `--kong-workspace` flags, as we previously had none.

RBAC users without super-admin rights are still "admin" users in the general sense that they interact with the admin API. KIC shouldn't need to alter how it behaves based on the permissions assigned to the provided user: the onus is on Kong's human administrators to create a user with the permissions KIC needs (generally super- or workspace-admin), and KIC should indeed fail if its user has insufficient permissions, since it can't possibly recover from that on its own.

Workspace creation failing if you provide a user without super-admin is fine--you'd need a fairly contrived configuration for that to occur: you're usually either providing a super-admin or workspace-admin user, and to create the latter at all you need to create the workspace first. You'd likely only encounter workspace creation failures due to permissions if you intentionally created a user with cluster permissions but specifically disabled workspace administration, which is just a misconfiguration, not something we need to actually handle.

Reply:
re-use existing interface 5ac1d90c5e6aa8319c4fd3169d6e45da4810e7cf